podcasts/PlaybackManager.swift

```diff
@@ -1945,7 +1945,7 @@ class PlaybackManager: ServerPlaybackDelegate {
     // MARK: - Private helpers
 
     private func checkIfStreamBufferRequired(episode: BaseEpisode, effects: PlaybackEffects) {
-        let downloadEpisode = effects.trimSilence.isEnabled() || FeatureFlag.cachePlayingEpisode.enabled
+        let downloadEpisode = effects.trimSilence.isEnabled() && !FeatureFlag.cachePlayingEpisode.enabled
```

Comment:
Maybe I'm seeing this wrong but if the flag is off the condition above will say true ( there is a !) so the download will happen if trim is enabled.
If the flag is on we don't want to do this download because we will use the download done by the export session.

Reply:
I missed the `!`, shame on me! Makes sense, thanks for explaining it!